models/common/fhir-path-reader.ts

```diff
@@ -0,0 +1,32 @@
+import {evaluate} from "fhirpath"
+import {Resource} from "fhir/r4"
+
+/*
+* Fhir Path Reader, used to extract values from FHIR.
+* see http://hl7.org/fhirpath/ for the specification
+* and https://github.com/HL7/fhirpath.js#readme
+* for the latest implementation
+*/
+
+export class FhirPathReader {
+    private fhir: Resource
+    constructor(fhir: Resource) {
+      this.fhir = fhir
+    }
+
+    read(path: string): string {
```

Comment:
are there times that read ever returns not a string? in FHIR/Json we can have numbers right?

Reply:
As mentioned, this would be a nice to have, if more classes will be added to handle additional FHIR resources